docs/install_linux_mac.rst

```diff
@@ -242,29 +242,19 @@ Installing Red
 
 Choose one of the following commands to install Red.
 
-.. note::
-
-    If you're not inside an activated virtual environment, include the ``--user`` flag with all
-    ``python3.8 -m pip install`` commands, like this:
-
-    .. code-block:: none
-
-        python3.8 -m pip install --user -U setuptools wheel
-        python3.8 -m pip install --user -U Red-DiscordBot
-
 To install without additional config backend support:
 
 .. code-block:: none
 
-    python3.8 -m pip install -U setuptools wheel
-    python3.8 -m pip install -U Red-DiscordBot
+    python -m pip install -U setuptools wheel
```

Comment:
Can we start updating `pip` here too? If we require venv, we can safely do so and it avoids issues with outdated pip which can be seen in #support.
(remember to update it in all 4 places)